src/scripts.js

```diff
@@ -285,14 +290,76 @@ function dropYear(dates) {
     userIDs.forEach(userID => {
       let userName = findUserName(Number(userID));
       list += `<li class="friends_li">
-             <p class="data-text"><b>${userName}</b>:</p>
-             <p class="data-text border-bottom">${activity.returnFriendsStepCount()[0][userID]} Steps</p>`;
+             <p class="data-text friends-step"><b>${userName}</b>:</p>
+             <p class="data-text friends-step border-bottom">${activity.returnFriendsStepCount()[0][userID]} Steps</p>`;
     });
     list += `</ul>`;
     return list;
   }
 
-  $('#friends-step').after(`${insertFriendSteps()}`);
+  $('#friends-data').text('Weekly Steps').after(`${insertFriendSteps()}`);
+
+  function insertFriendHydro() {
+    let list = `<ul class="friends_ul">`
+      userIDs.forEach(userID => {
+        let userName = findUserName(Number(userID));
+        const user = new User(userData[userID]);
+        const hydration = new Hydration(hydrationData, user);
+        list += `<li class="friends_li">
+             <p class="data-text"><b>${userName}</b>:</p>
+             <p class="data-text border-bottom">${hydration.returnFriendsDailyFluidOunces()} Ounces</p>`;
+      });
+      list += `</ul>`;
+      return list;
+    }
+
```

Comment:
These functions look like they could be refactored into one.

Reply:
Nope pull from classes and the timing gets messed up when its one function